app/services/form_service.py

```diff
@@ -0,0 +1,87 @@
+import http
+from bson import ObjectId
+from app.domain.database.db import Database
+from app.domain.models.form import FormModel
+from app.domain.models.dto.response import ResponseDTO
+from app.services.ong_service import OngService
+from app.services.animal_service import AnimalService
+
+class FormService:
+    def __init__(self,ong_service: OngService, animal_service: AnimalService):
+        self.db = Database()
+        self.ong_service = ong_service
+        self.animal_service = animal_service
+        self.form_collection = self.db.get_database().get_collection("forms")
+        
+    def create_form(self, ong_id:str, animal_id:str, form: FormModel):
+        try:
+            with self.db.session.start_transaction():
+                animals = self.ong_service.get_ong_animals(ong_id)
+                has_animal = False
+                for animal in animals:
+                    if str(animal.get("id")) == animal_id:
+                        has_animal = True
+                        break
+                if not has_animal:
+                    return ResponseDTO(http.HTTPStatus.BAD_REQUEST, "Animal doesn't belongs to ONG. Aborting.", None)
```

Comment:
Todos os responseDTO tem a ordem Data, Message, Status (acabei invertendo foi mal)
```suggestion
                    return ResponseDTO(None, "Animal doesn't belongs to ONG. Aborting.", http.HTTPStatus.BAD_REQUEST)
```